packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/useApprovalRequestActionsCell.tsx

```diff
@@ -91,7 +91,7 @@ export const useApprovalRequestActionsCell = (
     field: 'actions',
     renderHeader: () => null,
     sortable: false,
-    flex: 0.5,
+    flex: 0.6,
```

Comment:
Why did we increase this value by 0.1? Did the migration change the styles?

Reply:
Cells in v7 are not `flex` anymore.
I had to adjust it everywhere when the inner container relied on the outer container's `flex` style.

This particular example wasn't fixed properly. Thanks for the comment, I rewrote it.